docs/index.html

```diff
@@ -0,0 +1,126 @@
+        </div>
+        <div class="lang-card">
+          <div class="lang-card-title">Java</div>
+          <a class="lang-card-link" href="#">Quick Start Guide</a>
+          <a class="lang-card-link" href="#">gRPC Basics Tutorial</a>
+          <a class="lang-card-link" href="#">API Reference</a>
+        </div>
+        <div class="lang-card">
+          <div class="lang-card-title">Python</div>
+          <a class="lang-card-link" href="#">Quick Start Guide</a>
+          <a class="lang-card-link" href="#">gRPC Basics Tutorial</a>
+          <a class="lang-card-link" href="#">API Reference</a>
+        </div>
+      
+        <div class="lang-card">
+          <div class="lang-card-title">Go</div>
+          <a class="lang-card-link" href="#">Quick Start Guide</a>
+          <a class="lang-card-link" href="#">gRPC Basics Tutorial</a>
+          <a class="lang-card-link" href="#">API Reference</a>
+        </div>
+        <div class="lang-card">
+          <div class="lang-card-title">Ruby</div>
+          <a class="lang-card-link" href="#">Quick Start Guide</a>
+          <a class="lang-card-link" href="#">gRPC Basics Tutorial</a>
+          <a class="lang-card-link" href="#">API Reference</a>
+        </div>
+        <div class="lang-card">
+          <div class="lang-card-title">Node.js</div>
+          <a class="lang-card-link" href="#">Quick Start Guide</a>
+          <a class="lang-card-link" href="#">gRPC Basics Tutorial</a>
+          <a class="lang-card-link" href="#">API Reference</a>
+        </div>
+      
+        <div class="lang-card">
+          <div class="lang-card-title">C#</div>
+          <a class="lang-card-link" href="#">Quick Start Guide</a>
+          <a class="lang-card-link" href="#">gRPC Basics Tutorial</a>
+          <a class="lang-card-link" href="#">API Reference</a>
+        </div>
+        <div class="lang-card">
+          <div class="lang-card-title">Objective-C</div>
+          <a class="lang-card-link" href="#">Quick Start Guide</a>
+          <a class="lang-card-link" href="#">gRPC Basics Tutorial</a>
+          <a class="lang-card-link" href="#">API Reference</a>
+        </div>
+        <div class="lang-card">
+          <div class="lang-card-title">PHP</div>
+          <a class="lang-card-link" href="#">Quick Start Guide</a>
+          <a class="lang-card-link" href="#">gRPC Basics Tutorial</a>
+          <a class="lang-card-link" href="#">API Reference</a>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+  <hr class="section-divider"/>
+<div class="container">
+  <div class="row">
+    <div class="col-md-11 nofloat center-block">
+      <h2 class="col-sm-8 doc-header">gRPC by Use Cases</h2>
+      <h5 class="col-sm-8 ">gRPC for Last Mile of Computing</h5>
+      <div class="col-sm-10 use-description">
+        <p>gRPC helps generate libraries for iOS and Android as well as offers a JS library for use in browsers.  We are eager to see how it plays in IOT environment as well :-)</p>
```

Comment:
@zinuga could you suggest revised wording here? IMO we should not talk about browsers in this section yet.